content/chronograf/v1.9/administration/migrate-to-high-availability.md

```diff
@@ -11,21 +11,43 @@ menu:
 Use [`chronoctl`](/chronograf/v1.9/tools/chronoctl/) to migrate your Chronograf configuration store from BoltDB to a shared `etcd` data store used for Chronograf high-availability (HA) clusters.
 
 {{% note %}}
-**Note:**  Migrating Chronograf to a shared data source creates new source IDs for each resource.
+#### Update resource IDs
+Migrating Chronograf to a shared data source creates new source IDs for each resource.
 Update external links to Chronograf dashboards to reflect new source IDs.
 {{% /note %}}
 
 1. Stop the Chronograf server by killing the `chronograf` process.
 2. To prevent data loss, we **strongly recommend** that you back up your Chronograf data store before migrating to a Chronograf cluster.
 3. [Install and start etcd](/chronograf/v1.9/administration/create-high-availability/#install-and-start-etcd).
-4. Run the following command, specifying the local BoltDB file and the `etcd` endpoint beginning with `etcd://`. (We recommend adding the prefix `bolt://` to an absolute path. To specify a relative path to the BoltDB file, the prefix cannot be used.)
+4. Run the following command, specifying the local BoltDB file and the `etcd` endpoint beginning with `etcd://`.
+   (We recommend adding the prefix `bolt://` to an absolute path.
+   To specify a relative path to the BoltDB file, the prefix cannot be used.)
 
     ```sh
-      $ chronoctl migrate -f bolt:///path/to/chronograf-v1.db -t etcd://localhost:2379
+    chronoctl migrate \
+      --from bolt:///path/to/chronograf-v1.db \
+      --to etcd://localhost:2379
     ```
 
-    > **Note:**
-      If you have authentication on `etcd`, use the standard URI format to define a username and password. For example, `etcd://user:pass@localhost:2379`
+    ##### Provide etcd authentication credentials
+    If authentication is enabled on `etcd`, use the standard URI basic
+    authentication format to define a username and password. For example:
+
+    ```sh
+    etcd://username:password@localhost:2379
+    ```
+
+    ##### Provide etcd TLS credentials
+    If TLS is enabled on `etcd`, provide your TLS certificate credentials using
+    the following query parameters in your etcd URL:
+
+    - **cert**: Path to client certificate file or PEM file
+    - **key**: Path to client key file
+    - **ca**: Path to trusted CA certificates
+
+    ```sh
+    etcd://127.0.0.1:2379?cert=/tmp/client.crt&key=/tst/client.key&ca=/tst/ca.crt
```

Comment:
Do these slashes need to be escaped, I wonder? :thinking:

Reply:
These shouldn't need to be, but it depends on the shell the user is using I guess.